test/integration/features/provision_earlier_version.feature

```diff
@@ -0,0 +1,42 @@
+@minishift-only
+Feature: Provision an older major release
+  As a user I can provision an older major version of minishift
+  Scenario: Starting Minishift with v1.5.1
+    Given Minishift has state "Does Not Exist"
+     When executing "minishift start --openshift-version v1.5.1" succeeds
+     Then Minishift should have state "Running"
```

Comment:
We should assert the version as well. That's in fact the most important here. Nothing in your tests proves that 1.5.1 is provisioned. It might be the flag is actually ignored and the default gets provisioned. None of the following tests would show that.

You should use:

    $ minishift openshift version

It is important to actually verify/test the core assertion we are trying to make. I